backend/prisma/schema.prisma

```diff
@@ -18,3 +31,44 @@ model User {
   failedLogins Int     @default(0)
   verified     Boolean @default(false)
 }
+
+model Category {
```

Comment:
Do we need this table since all the categories fall under "Algorithms?" Or do we want to open it for future extensions?

Reply:
Let's leave it open for extension rather than have to perform a normalization down the road.